packages/noya-designsystem/src/utils/sketchPattern.ts

```diff
@@ -0,0 +1,69 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { FileMap } from 'noya-sketch-file';
+
+export const imgFileExtensions = ['png', 'jpeg', 'webp'];
+export const mimeTypes = imgFileExtensions.map(
+  (e) => `image/${e === 'ico' ? 'x-icon' : e}`,
+);
+
+export type SketchPattern = {
+  _class: 'pattern';
+  image?: Sketch.FileRef | Sketch.DataRef;
+  patternFillType: Sketch.PatternFillType;
+  patternTileScale: number;
+};
+
+function encodeImg(input: Uint8Array) {
```

Comment:
Since we have this in a separate file already, should we delete it here? Presumably we'll end up replacing this with CanvasKit API calls anyway, so not super important

Reply:
delete the encodeImg funcion? I just have it for preview for now. I'll delete it when we change to use CanvasKit

Reply:
Is this the same as what's in `utils/encodeImg.ts`? Or is it different?

Reply:
Oh i forgot about that file 😅 I don't think we need it